cms_helpers/cms_toolbars.py

```diff
@@ -27,9 +27,15 @@ def populate(self):
         position = self.get_item_position(current_page_menu)
 
         urls = self.get_title_extension_admin()
+
+        try:
```

Comment:
I would add an explicit check
```suggestion
        if hasattr(self.toolbar, 'edit_mode_active'):
```